### PR TITLE
Enhancement/5

### PR DESCRIPTION
**Pull request by SilviaPetruta**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
 let screenType = '';
+let priceForHotdog = 0, priceForPopcorn = 0, priceForDrink = 0;
 
-screenChoise.addEventListener('change', () => {
-     checkScreenType(screenChoise.value);
+screenChoice.addEventListener('change', () => {
+     checkScreenType(screenChoice.value);
 });
 
+for (let i = 0; i < hotdogRadioBtns.length; i++) {
```
Comment (nmunro) [inline src/index.js:14]: Would this work? `for(let btn of hotdogRadioBtns) { ... }`

**Review by DanGBat — APPROVED** [review]
Happy with the changes.
I'll try and work my functions in at this point.

File: src/index.js (hunk 1 above)
Comment (SilviaPetruta) [reply src/index.js:14]: Yeah, I think it will work, do I need to change it or it's ok like this?

Comment (nmunro) [reply src/index.js:14]: It's ok the way it is, just was thinking how it can be made simpler :)

**Pull request merged**